### PR TITLE
Make pausedDuration constant

### PR DESCRIPTION
**Pull request by NicholasDotSol**
closes: #510 

Pause duration will not change, therefore making this a read-only
constant makes sense

### CODE REVIEW

File: implementation/contracts/system/TBTCSystem.sol [hunk 1]
```diff
@@ -35,7 +35,7 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
 
     bool _initialized = false;
     uint256 pausedTimestamp;
-    uint256 pausedDuration = 10 days;
+    uint256 constant pausedDuration = 10 days;
```
Comment (nkuba) [inline implementation/contracts/system/TBTCSystem.sol]: Thoughts on moving it to `TBTCConstants` library?

Comment (NicholasDotSol) [reply implementation/contracts/system/TBTCSystem.sol]: >  moving it to TBTCConstants library

Leaning towards no. It's not really a TBTCConstant, it's a System constant that will only really be used once.

**Comment by nkuba** [comment]
@NicholasDotSol can we update this branch with master?

**Comment by NicholasDotSol** [comment]
done

**Comment by nkuba** [comment]
@NicholasDotSol we need to update it again 😢

**Comment by NicholasDotSol** [comment]
@nkuba updated

**Pull request merged**